examples/remote/client.rs

```diff
@@ -0,0 +1,111 @@
+//! A simple command line client that allows issuing queries to a remote Bevy
+//! app via the BRP.
+
+use std::process;
+
+use argh::FromArgs;
+use bevy::prelude::default;
+use bevy::remote::{BrpQuery, BrpRequest, DEFAULT_PORT};
+use http_body_util::BodyExt as _;
+use hyper::client::conn::http1;
+use hyper::header::HOST;
+use hyper::Request;
+use hyper_util::rt::TokioIo;
+use serde_json::Value;
+use tokio::net::TcpStream;
+use tokio::task;
+
+/// TODO
+#[derive(FromArgs)]
+struct Args {
+    /// the host to connect to
+    #[argh(option, default = "\"127.0.0.1\".to_owned()")]
+    host: String,
+    /// the port to connect to
+    #[argh(option, default = "DEFAULT_PORT")]
+    port: u16,
+    /// the full type names of the components to query for
+    #[argh(positional, greedy)]
+    components: Vec<String>,
```

Comment:
I think this is really good to include, but I'd put this in a loop so the user can send/receive multiple commands REPL style.

Reply:
This intentionally isn't a full client: it's just an example. In practice I think what 90% of people will want to use is a GUI, which is out of scope, so I didn't bother adding a lot of extra features. You can always use `curl` if you want to experiment with the complete BRP (maybe I should add an example invocation to the command line).

Reply:
That is fair, I do agree that a GUI version of this client would be more desirable (even if it's currently out of reach).

Reply:
It's not out of reach--you could use egui, or a Web UI, or even sickle. It wouldn't be able to be in tree for a while, though.